include/fmt/color.h

```diff
@@ -209,18 +209,103 @@ struct rgb {
   uint8_t b;
 };
 
-void vprint_rgb(rgb fd, string_view format, format_args args);
-void vprint_rgb(rgb fd, rgb bg, string_view format, format_args args);
+namespace internal {
+FMT_CONSTEXPR_DECL const char FOREGROUND_COLOR[] = "\x1b[38;2;000;000;000m";
+FMT_CONSTEXPR_DECL const char BACKGROUND_COLOR[] = "\x1b[48;2;000;000;000m";
```

Comment:
These should go to `internal::data`. Also you don't need the `000;000;000m` part because you construct it in `ansi_color_escape`.